libs/util/shared/src/lib/utils/atomic-operations.ts

```diff
@@ -54,12 +54,25 @@ export const mapLogo = (source: SourceWithUnknownProps) => {
   return logo ? getAsUrl(`/geonetwork${logo}`) : null
 }
 
+export const hydrateWithRecordLogo = (
```

Comment:
I would just call it `hydrateContactsWithRecordLogo`

Do the tests cover this function ? Probably

Reply:
Renamed. 

Yes this function is covered in both org-from-metadata and org from-groups classes.